code/modules/power/supermatter/supermatter.dm

```diff
@@ -635,6 +635,15 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 				removed.gases[/datum/gas/miasma][MOLES] -= consumed_miasma
 				matter_power += consumed_miasma * MIASMA_POWER_GAIN
 
+		//Let's say that the CO2 touches the SM surface and the radiation turns it into Pluoxium.
+		if(gas_comp[/datum/gas/carbon_dioxide])
+			var/carbon_dioxide_pp = env.return_pressure() * gas_comp[/datum/gas/carbon_dioxide]
+			var/consumed_carbon_dioxide = clamp(((carbon_dioxide_pp - CO2_CONSUMPTION_PP) / (carbon_dioxide_pp + CO2_PRESSURE_SCALING)), CO2_CONSUMPTION_RATIO_MIN, CO2_CONSUMPTION_RATIO_MAX)
+			consumed_carbon_dioxide *= gas_comp[/datum/gas/carbon_dioxide] * combined_gas
+			if(consumed_carbon_dioxide)
+				removed.gases[/datum/gas/carbon_dioxide][MOLES] -= consumed_carbon_dioxide
```

Comment:
make it also consume oxygen since it was like that in the old recipe

Reply:
I will explain my thought process on this but I can tweak it if you folks think it needs the O2 as balance.

Since we can't stack radiation on the SM to increase production, the simple vectors we have to balance the generation is pressure or power, we already have a lot of power related SM stuff so I decided to do pressure. Mostly for flavor.

The issue I had is that a high pressure or power CO2/O2 SM is finicky and just having O2 safely on SM takes some knowledge, or it will be a speedrun into delam.

Pluox was always the "Hey I understand how the N2 SM works, what can I do as a next step now?" suggestion so wanted to keep it that way without turning it into a Noob trap by forcing them to manage O2.

Reply:
Why not have the production scaled by EER?

Reply:
After toying more with the new CO2 changes, I decided to put O2 back in the reaction as an extra control.